fall18/p2/test.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+import subprocess, json, re, sys
+from inspect import getmembers, isfunction
+import main
+
+PASS = 'PASS'
+PROBLEMS = 20
+EPSILON = 0.0001
+
+TESTS = [
+    ("getMaximumLand", ["Wisconsin", "Iowa", "Minnesota"], 86935.83), #1-1
```

Comment:
Hmmm, there are really only 4 distinct tests here.  I have trouble imaging how a student could get a grade other than 100%, 75%, 50%, etc...

Reply:
Yes... If you think 4 tests are better I can try that.

Reply:
No, we need at least 10 real tests that exercise different things

Reply:
Right now, it feel like we only have 4 real tests